articles/cloud-storage/cs-how-install-cifs-ecs.md

```diff
@@ -94,10 +94,10 @@ CIFS-ECS offers a single caching option:
 
 11. Click **OK** to return to the *Settings* page.
 
-12. The rest of the fields on the *Settings* page enable you to configure the root directory, encryption and compression settings.
```

Comment:
Original was correct (enable not enables because you are referring to multiple fields)

Reply:
Yes you're right, apologies.